### PR TITLE
Fixed minor UI issues

### LINKED ISSUE #35: Minor UI Fixes
- [x] Mandatory fields should be marked with an asterisk symbol. Ref: https://github.com/bigbinary/neeto-runner-web/issues/1241
- [x] Rename `Inactive Members` to `Deactivated Members`.

### PR DESCRIPTION
**Pull request by Thejus-Paul**
Fixes #35 
- Replaced inactive with deactivated for member status.
- Added asterisks for required fields.

### CODE REVIEW

**Comment by Thejus-Paul** [comment]
@amaldinesh7 _a Please review.

File: src/components/AddMember.js [hunk 1]
```diff
@@ -86,7 +86,7 @@ const AddMember = ({
                   <div className="w-full space-y-4">
                     {selectedMember ? (
                       <Input
-                        label="Email(s)"
+                        label="Email(s)*"
```
Comment (amaldinesh7) [inline src/components/AddMember.js:89]: You can add the required prop for the Input component. No need to add the asterisk explicitly.

File: src/components/AddMember.js [hunk 2]
```diff
@@ -106,7 +106,7 @@ const AddMember = ({
                     )}
 
                     <Select
-                      label="Role"
+                      label="Role*"
```
Comment (amaldinesh7) [inline src/components/AddMember.js:109]: Same here

**Review by amaldinesh7 — CHANGES_REQUESTED** [review]
@Thejus-Paul _a Please check the comments.

**Review by amaldinesh7 — APPROVED** [review]

**Pull request merged**